types.ts

```diff
@@ -525,7 +525,7 @@ export interface CustomAuthParameter {
  *   let secretIdTemplateName = "secretId-" + context.invocationToken;
  *   let urlWithSecret = "/api/entities/{{" + secretIdTemplateName + "}}"
  *   let secretValueTemplateName = "secretValue-" + context.invocationToken;
- *   let secretHeader = 'Authorization  {{"' + secretValueTemplateName + '"}}';
+ *   let secretHeader = "Authorization  {{" + secretValueTemplateName + "}}";
```

Comment:
don't you need to use different quotes inside the string (single quotes) or escape?

Reply:
There's a chance we should be escaping somehow, but the extra quotes were making the code not work. FWIW, https://coda.github.io/packs-sdk/guides/advanced/authentication/#custom-tokens doesn't have them and your urlWithSecret here doesn't have them.

Here's some example code I wrote that worked:
```
      const authHeaders = {
        'x-client': '8100a127-10c2-4f01-a18f-28b1b550637f',
        'x-api-user': `{{userId-${context.invocationToken}}}`,
        'x-api-key': `{{apiToken-${context.invocationToken}}}`,
      }

      let tagResponse = await context.fetcher.fetch({
        method: "GET",
        url: "https://habitica.com/api/v3/tags",
        headers: authHeaders,
      });
```

Reply:
oh wait nvm i was reading this wrong lol my bad

Reply:
somehow i thought there were "s inside a pair of "s but there isnt. thanks!